tests/entities/test_trace.py

```diff
@@ -229,3 +237,43 @@ def test_trace_pandas_dataframe_columns():
         data=TraceData(),
     )
     assert Trace.pandas_dataframe_columns() == list(t.to_pandas_dataframe_row())
+
+
+@pytest.mark.parametrize(
+    ("span_type", "name", "expected"),
+    [
+        (None, None, ["run", "add_one_1", "add_one_2", "add_two", "multiply_by_two"]),
+        (SpanType.CHAIN, None, ["run"]),
+        (None, "add_two", ["add_two"]),
+        (None, re.compile("add.*"), ["add_one_1", "add_one_2", "add_two"]),
```

Comment:
```suggestion
        (None, re.compile(r"add.*"), ["add_one_1", "add_one_2", "add_two"]),
        (None, re.compile(r"_two$"), ["add_two"]),
```

Let's add one more test case for regular expression